ios/FluentUI/Vnext/Avatar/AvatarGroup.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+@objc public enum MSFAvatarGroupStyle: Int, CaseIterable {
+    case stack
+    case pile
+}
+//
+/// `MSFAvatarGroupState` defines properties in an AvatarGroup
+///
+/// `maxDisplayedAvatars`: Maximum number of avatars displayed
+///
+/// `overflowCount`: Number to be displayed in the overflow avatar
```

Comment:
Based on discussions we had with other teams who were confused about the relation of the maxDisplayedAvatars and overflowCount, can you provide more details on the documentation of these 2 properties?

I think the main topics I'd want to clarify are:
 - **maxDisplayedAvatars** caps the number of displayed avatars and shows the remaining ones that were not displayed in the overflow avatars.
 - **overflowCount** adds to the overflow count in case the calling code did not provide all the avatars, but still want to convey more items than just the remainder of the avatars that could not be displayed due to the maxDisplayedAvatars property.